purchase_return/README.rst

```diff
@@ -0,0 +1,3 @@
+===============
```

Comment:
The README file does not need to be added, it is automatically generated from the files in the 'readme' folder.